src/CommonLibrariesForNET/AuthenticationClient.cs

```diff
@@ -19,17 +19,17 @@ public class AuthenticationClient : IAuthenticationClient, IDisposable
         private const string TokenRequestEndpointUrl = "https://login.salesforce.com/services/oauth2/token";
         private readonly HttpClient _httpClient;
 
-        public AuthenticationClient()
-            : this(new HttpClient())
+        public AuthenticationClient(string apiVersion = "v36.0")
+            : this(new HttpClient(), apiVersion)
         {
         }
 
-        public AuthenticationClient(HttpClient httpClient)
+        public AuthenticationClient(HttpClient httpClient, string apiVersion)
```

Comment:
This will fail as the existing calls don't pass in a default `apiVersion` and consequently, it calls the other constructor. I'll fix this by adding a default to `"v36.0"`.